translate/src/modules/editor/components/EditorSettings.test.js

```diff
@@ -4,6 +4,28 @@ import sinon from 'sinon';
 
 import { EditorSettings, EditorSettingsDialog } from './EditorSettings';
 
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useContext: () => ({ code: 'en' }),
+}));
+
+jest.mock('~/modules/project/hooks', () => ({
+  useProject: () => ({ slug: 'test' }),
+}));
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: (selector) =>
+    selector({
+      user: {
+        isAuthenticated: true,
+        managerForLocales: ['en'],
+        translatorForLocales: ['en'],
+        translatorForProjects: { 'en-test': true },
```

Comment:
The last two lines are not technically necessary, because if the user is a manager, it's also a translator.